### PR TITLE
Propagate SSO properties to UserAccount

### PR DESCRIPTION
**Pull request by sbryzak**
This PR modifies both UserSignupController and MasterUserRecordController to propagate the UserID and AccountID values through to the UserAccount resource.

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Looks good but it seems that you missed the mur controller tests. I see only usersignup controller tests.

File: tools.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build tools
 // +build tools
```
Comment (mfrancisc) [inline tools.go:2]: probably we don't need this anymore, apparently was used for go version <= 1.16

**Review by mfrancisc — APPROVED** [review]

**Comment by mfrancisc** [comment]
/retest

infra

**Pull request merged**